webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
```

Comment:
- [ ] OPTIONAL: Webpack compiles with a warning:

![image](https://user-images.githubusercontent.com/57421823/172696321-0a752bf2-de02-4514-a1b7-155a9011e27a.png)

I suggest you check your webpack config so this does not happen on each compilation.

- [ ] OPTIONAL: You are requiring lots of unnecessary dependencies on package.json, making the app extra bulky:

![image](https://user-images.githubusercontent.com/57421823/172696716-df7c0c80-3194-4bd9-a0d9-fb1a4cd3eb4a.png)

I suggest you trim your list to only the required dependencies.